### PR TITLE
[bitnami/redis] Use common password manager to handle password

### PR DESCRIPTION
**Pull request by LeoColomb**
### Description of the change

Switch to the common helper `common.secrets.passwords.manage` for managing the password instead of the local undocumented helper.

### Benefits

* Streamline with the other charts
* Use a documented helper
* Benefit from the common helper features

### Applicable issues

The custom helper was introduced by https://github.com/bitnami/charts/pull/8486

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/redis/templates/_helpers.tpl [hunk 1]
```diff
@@ -222,34 +222,11 @@ Get the password key to be retrieved from Redis&reg; secret.
...
-{{- if or .Values.auth.enabled .Values.global.redis.password }}
-    {{- if not (empty .Values.global.redis.password) }}
-        {{- .Values.global.redis.password -}}
-    {{- else if not (empty .Values.auth.password) -}}
-        {{- .Values.auth.password -}}
-    {{- else -}}
-        {{- include "getValueFromSecret" (dict "Namespace" (include "common.names.namespace" .) "Name" (include "redis.secretName" .) "Length" 10 "Key" (include "redis.secretPasswordKey" .))  -}}
-    {{- end -}}
-{{- end -}}
+{{ include "common.secrets.passwords.manage" (dict "secret" (include "redis.secretName" .) "key" (include "redis.secretPasswordKey" .) "providedValues" (list "global.redis.password" "auth.password") "length" 10 "skipB64enc" true "skipQuote" true "context" $) }}
```
Comment (jotamartos) [inline bitnami/redis/templates/_helpers.tpl:229]:
```suggestion
{{- if or .Values.auth.enabled .Values.global.redis.password }}
    {{ include "common.secrets.passwords.manage" (dict "secret" (include "redis.secretName" .) "key" (include "redis.secretPasswordKey" .) "providedValues" (list "global.redis.password" "auth.password") "length" 16 "skipB64enc" true "skipQuote" true "context" $) }}
{{- end }}
```

The password was previously generated under that condition. Apart from that, the default length was 16

**Review by jotamartos — CHANGES_REQUESTED** [review]
Thanks for your contribution, please take a look at my comment.

File: bitnami/redis/templates/_helpers.tpl (hunk 1 above)
Comment (LeoColomb) [reply bitnami/redis/templates/_helpers.tpl]: Thanks for your review. Suggestion applied.
I still believed the default length was 10, though, with the argument when calling the previous function. https://github.com/bitnami/charts/pull/29376/files#diff-ddd93ad4c7925e478912f6a48d16e6ac7d562f34083126c97736d210470f7e04L250

Comment (jotamartos) [reply bitnami/redis/templates/_helpers.tpl]: You are right @LeoColomb, I didn't notice that and saw that the default length in the function was 16. Accepting the change now

**Review by jotamartos — APPROVED** [review]

**Pull request merged**